bark/generation.py

```diff
@@ -287,11 +288,11 @@ def load_codec_model(use_gpu=True, force_reload=False):
     device = _grab_best_device(use_gpu=use_gpu)
     if device == "mps":
         # encodec doesn't support mps
-        device = "cpu"
+        device = "cuda"
```

Comment:
```suggestion
        device = "cpu"
```

If user has 'mps' selected, then cuda is not available. This breaks bark for M1 chips.

Reply:
okay i will fix that too